### DISCUSSION TITLE
middleware/elasticsearch/user-guide/create/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1863] [Announcements]
# middleware/elasticsearch/user-guide/create/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/elasticsearch/user-guide/create/

**Comment by liyuerich** [comment]
现在还有安装环境检测，回检查operator是否装好

**Reply by liyuerich** [reply]
现在还有安装环境检测，会检查operator是否装好

**Comment by liyuerich** [comment]
冷数据节点启用后， 默认专用主节点自动启用。这个能介绍一下吗？

**Reply by Michelle951** [reply]
fixed by pull request: https://github.com/DaoCloud/DaoCloud-docs/pull/1884
